config/initializers/hacks/hyrax_file_sets_controller.rb

```diff
@@ -20,5 +20,32 @@ def after_update_response
         end
       end
     end
+
+    # rubocop:disable Metrics/MethodLength
+    def render_unavailable
+      s = SolrDocument.find(Hyrax.query_service.find_parents(resource: unavailable_presenter.to_model).first.id)
+      @presenter = "Hyrax::#{s["has_model_ssim"].first}Presenter".constantize.new(s, current_ability)
```

Comment:
Since `unavailable_presenter` sets `@presenter` for the previous `:unauthorized` message, does this also need to be conditional on tombstoned vs other unauthorized?